modules/primo/primo.install

```diff
@@ -17,6 +18,76 @@ function primo_schema() {
   return $schema;
 }
 
+/**
+ * Implements hook_install().
+ *
+ * Set default during installation.
+ */
+function primo_facetbrowser_install() {
```

Comment:
Am I missing something or how can this be a `hook_install()` implementation? The module is called `primo` - not `primo_facetbrowser`.